Makefile

```diff
@@ -22,7 +22,7 @@ help:
 >@${MESSAGE} print '      make test'
 >@${MESSAGE} print '          Do unit and module tests.'
 >@${MESSAGE} print ' '
->@${MESSAGE} print '      make test-[apiserver | countsserver | db | workflow | ]'
+>@${MESSAGE} print '      make test-[apiserver | cggnn | countsserver | db | workflow | ]'
```

Comment:
I'm a bit troubled by this help text ending with a |

Also, this help text is wrong as we discussed earlier. This help should also be included in the maintenance.md doc.

Reply:
Yes you are right!

Reply:
I added the new help text in #103